bin/main.ts

```diff
@@ -1,17 +1,49 @@
-import { App } from '@aws-cdk/core';
-import * as ssp from '@aws-quickstart/ssp-amazon-eks';
-import { MyFluentBitAddOn } from '../dist';
+import { App } from 'aws-cdk-lib';
+import { EksBlueprint } from '@aws-quickstart/eks-blueprints';
+import { SnykMonitorAddOn } from '../dist';
+import { pipeline } from '../ci/pipeline';
 
 const app = new App();
 
-ssp.EksBlueprint.builder()
-    .addOns(new ssp.MetricsServerAddOn)
-    .addOns(new ssp.ClusterAutoScalerAddOn)
-    .addOns(new ssp.addons.SSMAgentAddOn) // needed for AWS internal accounts only
-    .addOns(new ssp.SecretsStoreAddOn) // requires to support CSI Secrets
-     .addOns(new MyFluentBitAddOn({
-         cloudWatchRegion: 'us-east-2',
-         //licenseKeySecret: 'my-addon-license', // if you set it, make sure there is a secret named my-addon-license-key in the target region
-         namespace: 'my-addon-namespace'
-     }))
-     .build(app, 'my-extension-test-blueprint');
+// load required parameters from the environment and validate them
+const account = process.env.CDK_DEFAULT_ACCOUNT!; // e.g. 492635582501
+const region = process.env.CDK_DEFAULT_REGION! || 'us-east-1';
+const stackID = process.env.STACK_ID || 'snyk-monitor-eks-blueprints-addon';
```

Comment:
I don't remember off the top of my head how JavaScript / TypeScript handles a case where the environment variable is completely empty. It might throw an error on the left-side statement that prevents it from processing the right-side. 

Worth a check, probably fine though.

Reply:
I agree, L10 is suspicious because we expect the left side to be non-null. Fixed in [d723574](https://github.com/snyk-partners/snyk-monitor-eks-blueprints-addon/pull/1/commits/d72357473f3e2c32d5f7af71d4e7ca312469fe29).